internal/cephfs/volumemounter.go

```diff
@@ -45,24 +45,12 @@ var (
 	fusePidMap    = make(map[string]int)
 	fusePidMapMtx sync.Mutex
 
-	fusePidRx = regexp.MustCompile(`(?m)^ceph-fuse\[(.+)\]: starting fuse$`)
-
-	quotaSupport = []util.KernelVersion{
-		{
-			Version:      4,
-			PatchLevel:   17,
-			SubLevel:     0,
-			ExtraVersion: 0, Distribution: "",
-			Backport: false,
-		}, // standard 4.17+ versions
-		{
-			Version:      3,
-			PatchLevel:   10,
-			SubLevel:     0,
-			ExtraVersion: 1062,
-			Distribution: ".el7",
-			Backport:     true,
-		}, // RHEL-7.7
+	fusePidRx   = regexp.MustCompile(`(?m)^ceph-fuse\[(.+)\]: starting fuse$`)
+	versionInfo = map[string][]int{
+		"version":      {4, 3},
```

Comment:
Putting this in a map is really awkward. Kernel versions are integers, and they should be used like that. No need to workaround the linter warning. The linter should allow a KernelVersion type similar to a variable, if it does not, I'd consider the linter broken (and you could file a bug/rfe against it).